### PR TITLE
Adding convenience alias commands

### PR DESCRIPTION
**Pull request by loganwright**

### CODE REVIEW

**Comment by brightredchilli** [comment]
:ship:

**Comment by chow-land** [comment]
:sheep: it

File: git_cookbook.md [hunk 1]
```diff
@@ -20,3 +20,20 @@ function pr () {
...
+alias gs='git status'
+alias gb='git branch'
+alias gg='git log --graph --oneline —all'
+alias go='git checkout'
+alias gom='git checkout master'
+alias gob='git checkout -b'
+alias grim='git rebase --interactive master'
+alias gri='git rebase -i'
+alias gra='git rebase --abort'
+alias grc='git rebase —continue'
```
Comment (paulrolfe) [inline git_cookbook.md]: --continue I believe

Comment (loganwright) [inline git_cookbook.md]: Good catch, somehow in copying it did that thing where it combines `--` into `–`

**Pull request merged**